src/GovUk.Education.ExploreEducationStatistics.Common/Services/TimePeriodLabelFormatter.cs

```diff
@@ -64,7 +71,23 @@ private string FormatYearInternal(int year)
             throw new ArgumentOutOfRangeException();
         }
 
-        private static TimePeriodLabelFormatter FormatterFor(TimeIdentifier timeIdentifier,
+        private string FormatCsvYearInternal(int year)
+        {
+            if (YearRegex.Match(year.ToString()).Success)
+            {
+                return YearFormat switch
+                {
+                    Academic => $"{year}{(year + 1) % 100:D2}",
```

Comment:
Can both these cases share the same return statement as they have the same implementation? Might make it easier if we have new year types added.

Also, didn't know about the D2 - nice!